CHANGELOG

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Upcoming]
-
+]]]]]ENTRY #1
+## [0.17.0] — 2019-06-28
```

Comment:
<!--
  1 failure:  The latest date i...
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
  File: CHANGELOG;
  Line: 9;
-->
- :no_entry_sign: The latest date in the CHANGELOG is not today!


```suggestion
## [0.2.0] - 2019-07-26
```